.github/workflows/winget-submission.yml

```diff
@@ -0,0 +1,27 @@
+name: Submit to Winget Community Repo
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  winget:
+    name: Publish winget package
+    runs-on: windows-latest
+    steps:
+      - name: Submit package to Winget Community Repo
+        run: |
+
+          $packageId = "Vencord.Vesktop"
+          $gitToken = "${{ secrets.WINGET_PAT }}"
```

Comment:
isn't this kinda bad? I'm fairly certain best practice is instead using the env: option to pass the secret as an environment variable

Reply:
I don't think this matters, but I do prefer the environment variable way. I've quite literally just taken this from [Microsoft's own workflow](https://github.com/microsoft/devhome/blob/944975f98a173da6e03d0ae2cf49b23d861e454e/.github/workflows/winget-submission.yml), so.